tests/integration/utils.py

```diff
@@ -183,6 +191,23 @@ def escape_path_go(dependency):
         return dependency
 
 
+def get_request_session():
+    """
+    Get a request session with a retry option.
+
+    :return: the requests session
+    :rtype: requests.Session
+    """
+    session = requests.Session()
+    retry = Retry(
+        total=5, read=5, connect=5, backoff_factor=1.3, status_forcelist=(500, 502, 503, 504)
+    )
+    adapter = requests.adapters.HTTPAdapter(max_retries=retry)
+    session.mount("http://", adapter)
+    session.mount("https://", adapter)
+    return session
+
+
```

Comment:
I wonder if we could reuse Cachito's retrying functions instead of re-writing this here. Thoughts?

Reply:
Was approved from `@lcarva` and `@chmeliik`.